homework/index.js

```diff
@@ -29,18 +29,68 @@
     });
     return elem;
   }
-
+  //get repos
   function main(url) {
-    fetchJSON(url, (err, repositories) => {
+    fetchJSON(url, (err, data) => {
       const root = document.getElementById('root');
       if (err) {
         createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-        return;
+      } else {
+        //createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
```

Comment:
Prefer to remove the code instead of commenting it, it's easier to read and understand the code if there aren't comments with old code inside it.